.travis.yml

```diff
@@ -1,55 +1,55 @@
-# FTR: Configuration on https://travis-ci.org/vector-im/riot-android/settings
-#
-# - Build only if .travis.yml is present -> On
-# - Limit concurrent jobs                -> Off
-# - Build pushed branches                -> On (build the branch)
-# - Build pushed pull request            -> On (build the PR after auto-merge)
-#
-# - Auto cancel branch builds            -> On
-# - Auto cancel pull request builds      -> On
-
 language: android
+dist: trusty
+sudo: required
 jdk: oraclejdk8
-sudo: false
 
-notifications:
-  email: false
+env:
+  global:
+  - ANDROID_API_LEVEL=28
+  - ANDROID_BUILD_TOOLS_VERSION=28.0.3
+  - ANDROID_ABI=armeabi-v7a
 
 android:
   components:
-    # Uncomment the lines below if you want to
-    # use the latest revision of Android SDK Tools
     - tools
     - platform-tools
-
-    # The BuildTools version used by your project
-    - build-tools-28.0.3
-
-    # The SDK version used to compile your project
-    - android-28
+    - tools
```

Comment:
`tools` is listed twice.

Minor, but pushing a commit will help trigger a travis-ci build.

Reply:
Now it's listed only once.